lib/galaxy/tools/deps/resolvers/modules.py

```diff
@@ -31,11 +31,12 @@ def __init__(self, dependency_manager, **kwds):
         find_by = kwds.get('find_by', 'avail')
         prefetch = _string_as_bool(kwds.get('prefetch', DEFAULT_MODULE_PREFETCH))
         self.modulecmd = kwds.get('modulecmd', DEFAULT_MODULECMD_PATH)
+        self.default_indicator = kwds.get('default_indicator', DEFAULT_INDICATOR)
```

Comment:
if you're getting this from the xml file, would you add a (commented out? comment documented?) sample of how it might look? Most admins won't dig into the codebase to figure out a parameter's use